### PR TITLE
Add cpu binding

### PR DESCRIPTION
**Pull request by critical27**
I found lots of context switch during other test. So I try to set cpu affinity, but the sad conclusion is that whether binding cpu or not has very little impact on performance at least for now. The only good news is that, it do help a bit to reduce the context switch.

As for graphd, the bottleneck clearly is not here. I will do other test later. We need to find the bottleneck for storage using other stress tool.

### CODE REVIEW

**Comment by bright-starry-sky** [comment]
itc

File: src/common/cpu/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2020 vesoft inc. All rights reserved.
```
Comment (darionyaphet) [inline src/common/cpu/CMakeLists.txt:1]: 2021 ?

**Comment by cangfengzhs** [comment]
I have also encountered the bottleneck of the stress tool before, which looks like lock contention caused by the graphd session mechanism.

File: src/common/cpu/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+#
+# This source code is licensed under Apache 2.0 License.
+
+nebula_add_library(
+    cpu_obj OBJECT
+    CpuInfo.cpp
+    CpuBinder.cpp
+)
+
+nebula_add_subdirectory(test)
```
Comment (SuperYoko) [inline src/common/cpu/CMakeLists.txt:11]: New line at EOF.

**Comment by critical27** [comment]
Add later if necessary, close for now

**Pull request closed without merging**